Dockerfile.nodejs16-linux

```
@@ -12,15 +10,15 @@ COPY build/* /usr/local/bin/
 COPY yarn-source /yarn-source
 
 RUN setup-base.sh
+RUN GOVERSION=$(curl -s https://go.dev/VERSION?m=text | head -1 | sed 's/go//') \
```

Just for kicks, `head` isn’t necessary (and the `head -1` form is deprecated, `head -n 1` should be used instead):

```suggestion
RUN GOVERSION=$(curl -s https://go.dev/VERSION?m=text | sed '1s/go//;q') \
```